Mesh_3/examples/Mesh_3/remesh_polyhedral_surface.cpp

```diff
@@ -0,0 +1,60 @@
+#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
+
+#include <CGAL/Mesh_triangulation_3.h>
+#include <CGAL/Mesh_complex_3_in_triangulation_3.h>
+#include <CGAL/Mesh_criteria_3.h>
+
+#include <CGAL/Polyhedral_mesh_domain_with_features_3.h>
+#include <CGAL/make_mesh_3.h>
+
+// Domain 
+typedef CGAL::Exact_predicates_inexact_constructions_kernel K;
+typedef CGAL::Polyhedral_mesh_domain_with_features_3<K> Mesh_domain;
+
+// Polyhedron type
+typedef CGAL::Mesh_polyhedron_3<K>::type Polyhedron;
+
+// Triangulation
+typedef CGAL::Mesh_triangulation_3<Mesh_domain>::type Tr;
+typedef CGAL::Mesh_complex_3_in_triangulation_3<
+  Tr,Mesh_domain::Corner_index,Mesh_domain::Curve_segment_index> C3t3;
+
+// Criteria
+typedef CGAL::Mesh_criteria_3<Tr> Mesh_criteria;
+
+// To avoid verbose function and named parameters call
+using namespace CGAL::parameters;
+
+int main()
+{
+  // Load a polyhedron
+  Polyhedron poly;
+  std::ifstream input("data/lion-head.off");
+  input >> poly;
+  // Create a vector with only one element: the pointer to the polyhedron.
+  std::vector<Polyhedron*> poly_ptrs_vector(1, &poly);
+
+  // Create a polyhedral domain, with only one polyhedron,
+  // and no "bounding polyhedron": the volumic part of the domain will be
+  // empty.
+  Mesh_domain domain(poly_ptrs_vector.begin(), poly_ptrs_vector.end());
+  
+  // Get sharp features
+  domain.detect_features();
```

Comment:
I added a function `detect_borders()` which operates on the bounding polyhedron.
TODO:  The domain class states that the bounding polyhedron must be closed. We must add that it may have borders.